tests/Integration/UncaughtExceptionTest.php

```diff
@@ -27,8 +26,45 @@ public function testUncaughtExceptionIsLoggedAndThrown()
                 ]
             ]
         ]);
-        self::$kernel->boot();
 
-        $this->logHandler()->hasCriticalThatContains('An uncaught exception occurred, ESB will be stopped now!');
+        try {
+            $this->expectException(\Amp\Beanstalk\ConnectException::class);
+            self::$kernel->boot();
+        } finally {
+            $this->assertTrue($this->logHandler()->hasCriticalThatContains('An uncaught exception occurred, ESB will be stopped now!'));
```

Comment:
You're right! You also fixed this assertion that didn't execute because of the exception. Thank you!